.devcontainer/denv.sh

```diff
@@ -140,6 +140,28 @@ fi
 env="${env}\nCRTOOL=${CRTOOL}"
 [[ -n ${CRTOOL_DL} ]] && cr8="${cr8}\n${CRTOOL_DL}"
 ##############################
+CHR_VER="$(findVer 'set(laterChrome_VER' SWATFramework/CMakeLists.txt */defaults.txt)"
+if [[ -n "${CHR_VER}" ]] && ${doisrhub}; then
+  # TRICKY: installing with yum or rpm fails because of conflicts with minimum chrome
+  # so extract from rpm using rpm2cpio and only keep the ./opt/google directory
+  LATER_CHROME="echo 'repo_add_once=false' > /etc/default/google-chrome \
+    && mkdir -p /opt/browsers-later/extract \
+    && cd /opt/browsers-later/extract \
+    && rpm2cpio https://dl.google.com/linux/chrome/rpm/stable/$(uname -m)/google-chrome-stable-${CHR_VER}-1.$(uname -m).rpm | cpio -idmv \
+    && cd .. \
+    && mv extract/opt/google . \
+    && rm -rf extract/"
+fi
+env="${env}\nLATER_CHROME=${LATER_CHROME}"
+##############################
+FOX_VER="$(findVer 'set(laterFirefox_VER' SWATFramework/CMakeLists.txt */defaults.txt)"
+if [[ -n "${FOX_VER}" ]] && ${doisrhub}; then
+  LATER_FIREFOX="mkdir -p /opt/browsers-later \
```

Comment:
the later browsers are both installed to `/opt/browsers-later/`

here's a directory listing
```
smanders@bpbldlatest:/bpvol/buildpro (laterBrowsers)
$ ls /opt /opt/browsers-later/
/opt:
browsers-later  extern  firefox  google  rh

/opt/browsers-later/:
firefox  google

```